packages/custom-templated-path-webpack-plugin/test/fixtures/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+
+const { basename } = require( 'path' );
+
+/**
+ * Internal dependencies
+ */
+
+const CustomTemplatedPathPlugin = require( '../../' );
+
+module.exports = {
+	mode: 'development',
+	context: __dirname,
+	entry: './entry',
+	output: {
+		filename: '[basename].js',
+		path: __dirname,
+	},
+	plugins: [
+		new CustomTemplatedPathPlugin( {
+			basename( path, data ) {
+				console.log(data)
```

Comment:
I think it wasn’t expected to be landed 😃

Reply:
😞

Reply:
At least it's in a test file.

Reply:
I need to add this to jest-console 😎